src/sentry/auth/helper.py

```diff
@@ -295,13 +298,13 @@ def handle_attach_identity(
             )
 
         if member is None:
-            member = self._get_organization_member(auth_identity)
+            member = self._get_organization_member(auth_identity, user)
```

Comment:
i wrote this code and looking at it, it seems a better name is `_get_or_create_organization_member`